wire-ios/Wire-iOS/Sources/UserInterface/Conversation Options/SecureLinkHeaderCell.swift

```diff
@@ -0,0 +1,95 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import UIKit
+import WireCommonComponents
+
+final class SecureLinkHeaderCell: UITableViewCell, CellConfigurationConfigurable {
+
+    // MARK: - Properties
+
+    private let topSeparator = UIView()
+    private let iconImageView = UIImageView()
+    private let titleLabel = UILabel()
+    private let subtitleLabel = UILabel()
+
+    // MARK: - Init
+
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        setupViews()
+        createConstraints()
+    }
+
+    @available(*, unavailable)
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    private func setupViews() {
+        contentView.addSubview(topSeparator)
+        [iconImageView, titleLabel, subtitleLabel].forEach(contentView.addSubview)
+
+        iconImageView.contentMode = .scaleAspectFit
+        iconImageView.image = UIImage(resource: .shield)
+        iconImageView.tintColor = SemanticColors.Icon.foregroundDefaultBlack
+
+        titleLabel.font = UIFont.font(for: .footnoteSemibold)
+        titleLabel.text = L10n.Localizable.GuestRoom.SecureLink.Header.title
+
+        subtitleLabel.numberOfLines = 0
+        subtitleLabel.font = UIFont.font(for: .caption1)
+        subtitleLabel.text = L10n.Localizable.GuestRoom.SecureLink.Header.subtitle
+    }
+
+    private func createConstraints() {
+        [topSeparator, iconImageView, titleLabel, subtitleLabel].forEach { $0.translatesAutoresizingMaskIntoConstraints = false }
```

Comment:
Not sure if it makes a difference, but I always set `translatesAutoresizingMaskIntoConstraints = false` before adding the views as subviews. Do you know more about it?

Reply:
Nope I don't know if it makes a difference.